mt-st.bash_completion

```diff
@@ -22,8 +23,19 @@ _mt () {
             COMPREPLY=($(compgen -W "$devs" -- "$cur"))
             return
             ;;
```

Comment:
This section gets invoked if the current command looks like:
`mt -f /dev/nstX stsetoptions `